gnomad/gnomadv4_cost_run_extract_rows.py

```diff
@@ -4,6 +4,7 @@
 ## annotate gnomADv4 with frequencies and variant QC standard fields
 ## export the rows as a hail.Table on a cpg bucket
 
+from random import shuffle
```

Comment:
`shuffle` is a parameter name of `repartition`, therefore you don't need to import the module.

```suggestion
```

Reply:
Good point thanks Leo, this was added automatically by Visual Studio Code, I have to check these settings